test-junit5/src/main/java/io/micronaut/test/extensions/junit5/MicronautJunit5Extension.java

```diff
@@ -89,6 +95,20 @@ public void beforeAll(ExtensionContext extensionContext) throws Exception {
                 Object testInstance = extensionContext.getRequiredTestInstance();
                 applicationContext.inject(testInstance);
             }
+
+            Sql[] sqlAnnotations = testClass.getAnnotationsByType(Sql.class);
```

Comment:
I would use `specDefinition` to access `AnnotationValue` and modify `TestSqlAnnotationHandler` to recieve it instead of the annotation, which we don't use anywere.

Reply:
Done (I think) 😉